### PR TITLE
Updates the plugin for the 0.5 release

### PR DESCRIPTION
**Pull request by josh146**
**Description of the Change:**

* Fixes tests to ensure they are compatible with the new `wires.Any` and `wires.All` enumerations.

* Updates the version number

**Benefits:** n/a

**Possible Drawbacks:** n/a

**Related GitHub Issues:** n/a

### CODE REVIEW

File: tests/test_gaussian.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def test_unsupported_gates(self):
         for g in all_gates - gates:
             op = getattr(qml.ops, g)
 
-            if op.num_wires == 0:
+            if op.num_wires <= 0:
```
Comment (co9olguy) [inline tests/test_gaussian.py]: just curious, why are these changed?

Comment (josh146) [reply tests/test_gaussian.py]: Originally PennyLane operations either specified a specific number of required wires, or 'all wires' (`Operation.num_wires = 0`).

In practice though, this didn't work, as there existed operations that didn't fit into these two categories --- namely operations that work on 'any' wires (`QubitStateVector`, `BasisState`, `GaussianState`).

In https://github.com/XanaduAI/pennylane/pull/277, we created enumerations for `Any` and `All` wires to fix this. For backwards compatibility, `All == 0`, and `Any = -1`.

This test is marked for deprecation, so this is just a quick fix to ensure it continue to pass.

Comment (co9olguy) [reply tests/test_gaussian.py]: That's what I figured. :+1:
